batterymonitor@pdcurtis/files/batterymonitor@pdcurtis/3.2/applet.js

```diff
@@ -229,6 +224,21 @@ MyApplet.prototype = {
         }
     },
 
+    setup_icon_paths: function () {
+        this.battery100 = metadata.path + `/icons/${this.iconTheme}//battery-100.png`;
+        this.battery080 = metadata.path + `/icons/${this.iconTheme}//battery-080.png`;
+        this.battery060 = metadata.path + `/icons/${this.iconTheme}//battery-060.png`;
+        this.battery040 = metadata.path + `/icons/${this.iconTheme}//battery-040.png`;
+        this.batteryCaution = metadata.path + `/icons/${this.iconTheme}//battery-caution.png`;
+        this.batteryLow = metadata.path + `/icons/${this.iconTheme}//battery-low.png`;
+        this.batteryCharging100 = metadata.path + `/icons/${this.iconTheme}//battery-charging.png`;
+        this.batteryCharging080 = metadata.path + `/icons/${this.iconTheme}//battery-charging-080.png`;
+        this.batteryCharging060 = metadata.path + `/icons/${this.iconTheme}//battery-charging-060.png`;
+        this.batteryCharging040 = metadata.path + `/icons/${this.iconTheme}//battery-charging-040.png`;
+        this.batteryChargingCaution = metadata.path + `/icons/${this.iconTheme}//battery-charging-caution.png`;
+        this.batteryChargingLow = metadata.path + `/icons/${this.iconTheme}//battery-charging-low.png`;
```

Comment:
It looks like there are double slashes here. This will still work but with the other changes, this is a quick edit.

Reply:
fixed